src/Lib/2D/CachedFontRenderer.cpp

```diff
@@ -124,6 +126,8 @@ int main(int argc, char *argv[]) {
   Package::assignDataDir();
   filesystem::addToSearchPath(Package::getDataDir().c_str());
 
+  MenuConfig::menu_font = new std::string("GNUUnifont9FullHintInstrUCSUR.ttf");
```

Comment:
shouldn't this come from the lua script now?

Reply:
Depends. If we're treating this like a full blown unit test, then this is probably better and this hardcoded string is a stub to substitute the lua-calling code. But if we want to test it more as part of the system, then calling the lua would be better.